bigbluebutton-html5/imports/ui/components/breakout-join-confirmation/container.jsx

```diff
@@ -31,6 +35,8 @@ export default withTracker(({ breakout, mountModal, breakoutName }) => {
     requestJoinURL(breakoutId);
   }
 
+  const currentVoiceUser = VoiceUsers.findOne({ meetingId: Auth.meetingID, intId: Auth.userID });
```

Comment:
```suggestion
  const currentVoiceUser = VoiceUsers.findOne({ meetingId: Auth.meetingID, intId: Auth.userID }, {fields: {joined: 1}});
```

Or get the joined flag via other means, maybe via AudioManager?

Reply:
Leaning towards using AudioManager